cmd/security-secretstore-setup/main.go

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Alain Pulluelo, ForgeRock AS
+ * @author: Tingyu Zeng, Dell
+ *
+ *******************************************************************************/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		usage.HelpCallbackSecuritySecretStore()
+	}
+
+	var initNeeded bool
+	var insecureSkipVerify bool
+	var configFileLocation string
+	var waitInterval int
+	var useProfile string
+	var useRegistry bool
+
+	flag.BoolVar(&initNeeded, "init", false, "run init procedure for security service.")
```

Comment:
@tingyuz can we queue up a discussion/review of the CLI interface for security-secretstore-setup like we've had for security-secrets-setup for next week? Not for this PR should be done before Fuji

Reply:
Sure we will put it into agenda.